### PR TITLE
Adding GetDefaultDurability()

### PR DESCRIPTION
**Pull request by Jesus-QC**
This will save some code to a few plugins.

### CODE REVIEW

**Comment by Jesus-QC** [comment]
Fuck, this is master branch, sorry for that <3

**Review by iRebbok — CHANGES_REQUESTED** [review]
It's actually better to get the values from the game directly, also they shouldn't be `int` either.

**Comment by galaxy119** [comment]
They should be uint, and there's a base-game function for this somewhere, use that, because these values could change in the future.

File: Exiled.API/Extensions/Item.cs [hunk 1]
```diff
@@ -308,5 +308,15 @@ public static void SetOther(this Player player, Inventory.SyncItemInfo weapon, O
...
         }
+
+        /// <summary>
+        /// Gets the default durability of any item.
+        /// </summary>
+        /// <param name="item">The <see cref="ItemType">item</see> that you want to get durability of.</param>
+        /// <returns>Returns the item durability.</returns>
+        public static float GetDefaultDurability(ItemType item)
+        {
+            return Pickup.Inv.GetItemByID(item).durability;
```
Comment (iRebbok) [inline Exiled.API/Extensions/Item.cs:319]: Make it not throw an exception when `Pickup.Inv.GetItemByID(item)` returned null

**Review by iRebbok — CHANGES_REQUESTED** [review]

**Review by iRebbok — APPROVED** [review]

File: Exiled.API/Extensions/Item.cs [hunk 2]
```diff
@@ -316,14 +316,7 @@ public static void SetOther(this Player player, Inventory.SyncItemInfo weapon, O
         /// <returns>Returns the item durability.</returns>
         public static float GetDefaultDurability(ItemType item)
```
Comment (BuildBoy12) [inline Exiled.API/Extensions/Item.cs:317]: Shouldn't it be `this ItemType item` so that it may be used as an extension?

Comment (iopietro) [reply Exiled.API/Extensions/Item.cs:317]: Yes, I agree.

**Review by BuildBoy12 — APPROVED** [review]

**Review by iopietro — APPROVED** [review]

**Pull request merged**